03/app.js

```diff
@@ -43,33 +43,41 @@ books.getAuthor = function(isbn) {
 }
 
 books.getTitle = function(isbn, lang) {
-    
-    const title = this[isbn]['title'];
 
-    if (title[lang]) {
+    if (typeof this[isbn] !== 'undefined' && typeof this[isbn]['title'] !== 'undefined') {
+        const title = this[isbn]['title'];
 
-        return title[lang];
+        if (typeof this[lang] !== undefined) {
```

Comment:
Pamiętaj, że jak jest `typeof` to zawsze on zwraca string więc powinno być `'undefined'` (u ciebie nie ma cudzysłowu)